### PR TITLE
Update Templates

### PR DESCRIPTION
**Pull request by KevinRansom**
The inbox templates used an incorrect mechanism to locate fsharp.core.dll.

The inbox templates used an early version of System.ValueTuple that did not support Portable 47 profile.

Both of these issues are fixed in this PR.

Kevin

### CODE REVIEW

File: src/fsharp/FSharp.LanguageService.Compiler/FSharp.LanguageService.Compiler.fsproj [hunk 1]
```diff
@@ -596,7 +596,7 @@
     <Reference Include="Microsoft.DiaSymReader"><HintPath>..\..\..\packages\Microsoft.DiaSymReader.1.0.8\lib\portable-net45+win8\Microsoft.DiaSymReader.dll</HintPath></Reference>
     <Reference Include="System.Reflection.Metadata"><HintPath>..\..\..\packages\System.Reflection.Metadata.1.4.1-beta-24227-04\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath></Reference>
     <Reference Include="System.Collections.Immutable"><HintPath>..\..\..\packages\System.Collections.Immutable.1.2.0\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath></Reference>
-    <Reference Include="System.ValueTuple"><HintPath>..\..\..\packages\System.ValueTuple.4.0.0-rc3-24212-01\lib\netstandard1.1\System.ValueTuple.dll</HintPath></Reference>
+    <Reference Include="System.ValueTuple"><HintPath>..\..\..\packages\System.ValueTuple.4.4.0-beta-24631-01\lib\netstandard10\System.ValueTuple.dll</HintPath></Reference>
```
Comment (brettfo) [inline src/fsharp/FSharp.LanguageService.Compiler/FSharp.LanguageService.Compiler.fsproj]: `netstandard1.0` instead of `netstandard10`

**Comment by brettfo** [comment]
Looks good pending one comment.

:shipit:

**Comment by KevinRansom** [comment]
@dotnet-bot please test this

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Pull request merged**